Podfile

```diff
@@ -11,9 +11,9 @@ install! 'cocoapods',
          :deterministic_uuids => false
 
 def common_pods
-  pod 'Adyen', '3.7.0'
-#  pod 'KarhooSDK', :git => 'git@github.com:karhoo/karhoo-ios-sdk.git', :branch => 'develop'
-  pod 'KarhooSDK', '1.5.5'
+  # pod 'Adyen', '3.7.0'
+ pod 'KarhooSDK', :git => 'git@github.com:karhoo/karhoo-ios-sdk.git', :branch => 'develop'
```

Comment:
Shouldn't we be pointing to the NSDK adyen upgrade branch for a full picture when testing?